templates/packages/order_edit.html

```diff
@@ -0,0 +1,254 @@
+                                        </form>
+                                    </li>
+                                </ul>
+                            {%endif%}
+                        </p>
+                    </div>
+                </div>
+            </div>
+        {%endfor%}
+    </div>
+    <style>
+        input::-webkit-outer-spin-button,
+        input::-webkit-inner-spin-button {
+           -webkit-appearance: none;
+           margin: 0;
+        }
+        input[type="number"] {
+           -moz-appearance: textfield;
+        }
+    </style>
+    <div class="modal fade" id="package-modal" tabindex="-1" data-bs-backdrop="static" data-bs-keyboard="false" role="dialog" aria-labelledby="package-modal-title" aria-hidden="true" dir="ltr">
+        <div class="modal-dialog modal-dialog-centered">
+            <div class="modal-content">
+                <div class="modal-header" dir="rtl">
+                    <div class="container">
+                        <div class="row">
+                            <div class="col-auto">
+                                <h5 class="modal-title" id="package-modal-title">יצירה/עריכה של חבילה</h5>
+                            </div>
+                            <div class="col">
+
+                            </div>
+                            <div class="col-auto">
+                                <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="modal-body">
+                    <div class="container-fluid">
+                        <form id="package-form" method="post"> {%csrf_token%}
+                            <input id="package-id" type="hidden" name="package_id"/>
+                            <label for="origin" class="form-label">Origin</label>
+                            <div id="origin" class="row g-1 mb-3">
+                                <div class="col-4">
+                                    <input id="origin-street" type="text" class="form-control rounded-0 border-0" name="origin_street" list="streets" placeholder="Street"/>
+                                </div>
+                                <div class="col-2">
+                                    <input id="origin-street-number" type="number" class="form-control rounded-0 border-0" name="origin_street_number" placeholder="0"/>
+                                </div>
+                                <div class="col-3">
+                                    <input id="origin-city" type="text" class="form-control rounded-0 border-0" name="origin_city" list="cities" placeholder="City"/>
+                                </div>
+                                <div class="col-3">
+                                    <select id="origin-area" class="form-select rounded-0 border-0" name="origin_area">

```

Comment:
Should be filled in automatically if the user selected a city.
Honestly, does area matter that much? if so, prompt to fill it only if the city is unknown because now I have to know where a city is even if I do autocomplete.

Reply:
I agree completely, and I did think about that when I first added this page, but now I think I know how to do it with javascript so... I'll try I guess

Reply:
WAIT

Reply:
Now I am really gravitating to that good autocomplete idea with some location API, that will mitigate the need for this (I bet we can retrieve the area/district from this api or just leave it blank for places that the API didn't autocomplete since it's not that important)

Reply:
Now I am really gravitating to that good autocomplete idea with some location API, that will mitigate the need for this (I bet we can retrieve the area/district from this api or just leave it blank for places that the API didn't autocomplete since it's not that important)

Reply:
Do it in some other branch. Investigate different ways and see what works best.